index.html

```diff
@@ -15,12 +15,27 @@
 </head>
 
 <body>
-
-  
-     
+      <nav>
+        <div>
+          <h1>Fun Bus</h1>
+        </div>
+        <div>  
+        <a>Home</a>
```

Comment:
& Youre anchor tags would want to LINK somewhere right? As a user, if we click a button that says "Home" we expect it to link somewhere-- home! (:

You can do so by using the `href` attribute that comes with anchor tags! `<a href="#">Home</a>` !
Then substitue # with the URL or the file you want to direct to! (: 👍